python/servo/package_commands.py

```diff
@@ -442,7 +460,9 @@ def update_brew(package, timestamp):
 
             brew_version = timestamp.strftime('%Y.%m.%d')
 
-            with tempfile.TemporaryDirectory(prefix='homebrew-servo') as tmp_dir:
+            with TemporaryDirectory(prefix='homebrew-servo') as tmp_dir:
+                print(tmp_dir)
```

Comment:
Is this intentionally left in, or was it for debugging output? Either with fix or left if it was intentional, r=me.